app/javascript/components/upload/UploadWizard.js

```diff
@@ -340,20 +340,20 @@ export function RawUploadWizard({ studyAccession, name }) {
     <StudyContext.Provider value={studyObj}>
       <div className="upload-wizard-react">
         <div className="row">
-          <div className="col-md-12 wizard-top-bar">
-            <span>{serverState?.study?.name}</span> / &nbsp;
-            <a href={`/single_cell/study/${studyAccession}`}>View study</a>
+          <div className="col-md-12 wizard-top-bar no-wrap-ellipsis">
+          <a href={`/single_cell/study/${studyAccession}`}>View study</a> / &nbsp;
+            <span >{serverState?.study?.name}</span>
```

Comment:
Helpful for accessibility:

```suggestion
            <span title="{serverState?.study?.name}">{serverState?.study?.name}</span>
```